snap/snapcraft.yaml

```diff
@@ -32,6 +32,8 @@ parts:
     plugin: go
     source: . 
     go-importpath: github.com/CanonicalLtd/UCWifiConnect 
+    build-packages:
+      - git
```

Comment:
why is git needed? if it is not essential, let's remove this to keep build times as short as possible